builtin/game/item_entity.lua

```diff
@@ -145,11 +145,17 @@ core.register_entity(":__builtin:item", {
 			y = pos.y + self.object:get_properties().collisionbox[2] - 0.05,
 			z = pos.z
 		})
+		-- Delete in 'ignore' nodes
+		if node and node.name == "ignore" then
```

Comment:
@paramat, afaIk, get_node_or_nil returns nil for ignore, you may want to use `if not node then`

Reply:
@HybridDog it only returns `nil` when the mapblock isn't loaded yet. If it is, `ignore` may be returned too.

Reply:
SmallJoker is correct, it's a common misunderstanding.

Reply:
You're right:
![screenshot_20180212_084140](https://user-images.githubusercontent.com/3192173/36087145-0a27aa9a-0fd1-11e8-9ad6-3ed9781d9248.png)
If you move further outside the boundaries, it returns nil:
![screenshot_20180212_084323](https://user-images.githubusercontent.com/3192173/36087157-1de5ae60-0fd1-11e8-9111-496746a5b70b.png)

Reply:
Yes each generated mapchunk at world edge has a 1 mapblock thick padding of ignore nodes around it, outside that is unloaded area.